dev-python/statsmodels/statsmodels-0.10.2.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} )
+
+inherit distutils-r1 eutils
+
+DESCRIPTION="Statistical computations and models for use with SciPy"
+HOMEPAGE="https://www.statsmodels.org/stable/index.html"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="doc examples test"
+
+COMMON_DEPEND="
```

Comment:
Looks like you dont need COMMON_DEPEND here. You can just name this into RDEPEND and then inherit that in DEPEND.

COMMON_DEPEND is used when RDEPEND and DEPEND have some similar dependencies, but some different as well.

Reply:
done